pkg/apis/pipeline/v1beta1/pipeline_validation.go

```diff
@@ -252,16 +252,20 @@ func filter(arr []string, cond func(string) bool) []string {
 func validatePipelineResults(results []PipelineResult) (errs *apis.FieldError) {
 	for idx, result := range results {
 		expressions, ok := GetVarSubstitutionExpressionsForPipelineResult(result)
-		if ok {
-			if LooksLikeContainsResultRefs(expressions) {
-				expressions = filter(expressions, looksLikeResultRef)
-				resultRefs := NewResultRefs(expressions)
-				if len(expressions) != len(resultRefs) {
-					errs = errs.Also(apis.ErrInvalidValue(fmt.Sprintf("expected all of the expressions %v to be result expressions but only %v were", expressions, resultRefs),
-						"value").ViaFieldIndex("results", idx))
-				}
+		if !ok {
+			errs = errs.Also(apis.ErrInvalidValue(("expected pipeline results to be task result expressions but no expressions were found"),
```

Comment:
nit: remove extra parenthesis
```suggestion
			errs = errs.Also(apis.ErrInvalidValue("expected pipeline results to be task result expressions but no expressions were found",
```

Reply:
done